Source/WebCore/platform/graphics/ImageBuffer.h

```diff
@@ -171,6 +171,8 @@ class ImageBuffer : public ThreadSafeRefCounted<ImageBuffer>, public CanMakeWeak
     WEBCORE_EXPORT virtual RefPtr<NativeImage> sinkIntoNativeImage();
     WEBCORE_EXPORT RefPtr<Image> copyImage(BackingStoreCopy = CopyBackingStore, PreserveResolution = PreserveResolution::No) const;
     WEBCORE_EXPORT virtual RefPtr<Image> filteredImage(Filter&);
+    RefPtr<Image> filteredImage(Filter&, std::function<void(GraphicsContext&)> drawCallback);
```

Comment:
Can the Filter& be const?

Reply:
Yes it can. I will clean this issue in a separate patch.